app/assets/stylesheets/retros.scss

```diff
@@ -49,5 +49,25 @@ html {
         }
       }
     }
+
+    #Boards {
```

Comment:
I recommend using the same naming convention(if any, or pick one) in all places, this is capitalized instead of all lowercase.